Taskfile.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+
+tasks:
+  macos_install_deps:
+    cmds:
+      - brew install kubeconform kube-score
+  update_version:
+    cmds:
+      - grep version charts/replicated-library/Chart.yaml | cut -d' ' -f2 > ./library_chart_version
+      - cp test-charts/test-replicated-library/Chart.yaml.tmpl test-charts/test-replicated-library/Chart.yaml
+      - 'sed -i.bak "s/^  version:.*$/  version: $(cat ./library_chart_version)/" test-charts/test-replicated-library/Chart.yaml'
+      - rm -f test-charts/test-replicated-library/Chart.yaml.bak
+      - rm -f ./library_chart_version
+  test:
+    cmds:
+      # Make sure we have a clean slate (for local use)
+      - rm -rf charts
```

Comment:
This command seems very dangerous. Running this command from the root folder would remove all of a users work in progress that hasn't been committed.

Reply:
This refers to the directory `test-charts/<testchartname>/charts`, not the repo root `charts`.

Specifically due to this line: `dir: test-charts/{{.CHART}}`

Which technically makes it safe :)